### PR TITLE
Ad 1.19 release notes

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Comment by evilnick** [comment]
These will get added at the top of the main release notes page too.

File: pages/k8s/1.19/release-notes.md [hunk 1]
```diff
@@ -30,7 +30,56 @@ Before upgrading, please read the [upgrade notes](/kubernetes/docs/upgrade-notes
...
+- Ubuntu 20.04
+
+The default operating system for deployed machines is now Ubuntu 20.04 (Focal). Ubuntu 18.04 (Bionic) and 16.04 (Xenial) are still supported.
+
+- SR-IOV CNI
+
+A new SR-IOV CNI addon has been made available for Charmed Kubernetes. Using
+SR-IOV CNI, it is now possible to take network interfaces that are SR-IOV
+Virtual Functions and attach them directly to pods. For more information, see
+the new [SR-IOV CNI documentation][cni-sriov].
```
Comment (Cynerva) [inline pages/k8s/1.19/release-notes.md:72]: The SR-IOV CNI documentation doesn't exist yet in this PR, but is added in a separate one: https://github.com/charmed-kubernetes/kubernetes-docs/pull/454

**Comment by kwmonroe** [comment]
We should remember to link https://ubuntu.com/kubernetes/docs/release-notes to this 1.19/release-notes page

File: pages/k8s/1.19/release-notes.md [hunk 2]
```diff
@@ -30,7 +30,69 @@ Before upgrading, please read the [upgrade notes](/kubernetes/docs/upgrade-notes
...
+secrets. If needed, any existing entries in previous authentication files
+(`basic_auth.csv` and `known_tokens.csv`) are migrated to secrets during the
+`kubernetes-master` charm upgrade.
+
+More information about this new service can be found in the
+[Authorisation and Authentication][authn] documentation.
+
+- New Calico configuration options
+  
+The new `veth-mtu` setting allows fine tuning of the MTU setting for optimum
```
Comment (evilnick) [inline pages/k8s/1.19/release-notes.md:64]: @tvansteenburgh I filled these out for you, please check

**Pull request merged**